releasenotes/notes/submodule-update-a2f6c5e9cb597a99.yaml

```diff
@@ -0,0 +1,16 @@
+---
+critical:
+  - |
+    In the newton and newton-rc branches openstack-ansible is referenced as a
```

Comment:
The branch references here aren't relevant for a release note which is aimed at operators, not developers. They make use of releases, so the reno should be describing fro an operator standpoint.

Something like 'From this release on, a rcbops fork is used for the openstack-ansible submodule. Due to this change, a pre-existing checkout of rpc-openstack will require a submodule sync to ensure that the subodule is pointing to the right repository. If this is not done, ... (explain the symptom which a operator would see). To resolve this execute the following ...'

Reply:
Done